index.xml

```diff
@@ -93,21 +93,33 @@ and only when, they appear in all capitals, as shown here.
 The following paragraphs are additions to RFC 6838.
     </t>
     <t>
-Media types MAY be registered with more than one suffix appended to the base
-subtype name. The suffixes MUST be interpreted as ordered. Valid media type
-names containing a structured suffix are built from right to left (not left
-to right). Characters on the left-most side of the left-most "+" in a subtype
-name specify the base subtype name. Characters to the right of each "+" in
-a subtype name denote additional structured syntax suffixes.
+Media types MAY be registered with more than one structured suffix appended to
+the base subtype name. Characters on the left-most side of the left-most "+" in
+a subtype name specify the base subtype name. The entire structured suffix is
+all of the characters to the right of the first "+" sign in the media type,
+including the initial "+" sign itself. The entire structured suffix MAY be
```

Comment:
Consistency throughout is vital to comprehension. I can make this into a PR, if that's better for you.
```suggestion
all of the characters to the right of the left-most "+" sign in the media type,
including the left-most "+" sign itself. The entire structured suffix MAY be
```